hw2/04-background.js

```diff
@@ -1 +1,55 @@
-// Add your code here
+// Set the default background color here cuz why not. I could do it in the html
+// but we're doing most of the work here anyways.
+document.addEventListener("DOMContentLoaded", () => {
+    document.body.style.backgroundColor = "rgb(108, 97, 219)";
+});
+
+// used to stop the interval
+const intervalID = [];
+
+// add a click listener to the button
+document.getElementById("actionButton").addEventListener("click", buttonListen);
+
+// Neatly generate the string for the random color to modify the style
+const randColor = function getRandomColor() {
```

Comment:
Had no idea this getRandomColor() function existed. I need to look at the libraries more.  I'm so used to building everything from scratch.

Reply:
It's not, I just wrote that function and gave it a const shortened name to give it structured hoisting.